.changeset/sharp-lobsters-build.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/backend-plugin-api': minor
```

Comment:
I think we could get away with calling this a patch since we're not breaking non-broken behavior, and all of the type renames are for output types.

Reply:
alright, that makes sense! nerfin' and merjin'